api/controllers/main.js

```diff
@@ -34,4 +34,21 @@ module.exports = {
 
     res.render('index.html', context);
   },
+
+  robots(req, res) {
+    const PROD_CONTENT = 'User-Agent: *\nDisallow: /preview\n';
+    const DENY_ALL_CONTENT = 'User-Agent: *\nDisallow: /\n';
+
+    res.set('Content-Type', 'text/plain');
+
+    // If this is the production instance and the request came to the production hostname
+    if (config.app.app_env === 'production' &&
+      config.app.hostname === `https://${req.hostname}`) {
```

Comment:
`req.protocol` is always `http` when deployed to cloud.gov since http is used for internal routing within cloud.gov, but our `APP_ENV` environment variable is always `https` since that is what is needed for external clients. Thus, I've hardcoded the protocol as `https://` here in this if-condition.